python-sdk/tests/databases/test_snowflake.py

```diff
@@ -278,12 +278,12 @@ def test_load_file_to_table_natively_for_fallback(
     )
     database, target_table = database_table_fixture
     filepath = str(pathlib.Path(CWD.parent, "data/sample.csv"))
-    response = database.load_file_to_table_natively_with_fallback(
-        source_file=File(filepath),
-        target_table=target_table,
-        enable_native_fallback=False,
-    )
-    assert response is None
+    with pytest.raises(DatabaseCustomError):
```

Comment:
@tatiana Should we also have a test where `load_file_to_table_natively_with_fallback` don't raise an exception? 

Also, we can verify the kind of exception we are getting for this file? like substring-based check?

Reply:
@utkarsharma2 there is a test in `test_bigquery`, which was also updated, which has the scenario in which we don't raise an exception.

In regards to the exception, at the moment the code does not raise any meaningful message to the users. SInce this is a bug fix because our master branch is broken, I don't think we should be doing further changes in this PR - but we can look into improving the user experience  in these scenarios in separate tickets/PRs.

Reply:
Sure @tatiana, we can address that in a separate PR.